Justfile

```diff
@@ -1,6 +1,7 @@
 # Big idea behind using a Justfile is so that we can have modules like in sbt.
 
 besom-version := `cat version.txt`
+besom-cfg-version := `cat besom-cfg/version.txt`
```

Comment:
I assume there is a reason behind versioning cfg separately, but wouldn't it make sense to align the versions, just for the sake of simplicity?

Reply:
no, it wouldn't, this stuff has and has to have a completely separate lifecycle